source/spelling_wordlist.txt

```diff
@@ -5,12 +5,14 @@
 accessor
 accessors
 backend
+backends
 destructor
 atomics
 dimensionality
 runtime
 runtimes
 subdevices
+sycl
```

Comment:
Now we can remove lowercase sycl from here

Reply:
Fixed with : https://github.com/imvucic/SYCL_Reference/pull/4/commits/7e28b14ffc6e01a4e7e4e8886323a270912b3de6